js/packages/image/package.json

```
@@ -0,0 +1,72 @@
+{
+  "name": "@applitools/image",
+  "version": "1.0.0",
+  "description": "Image primitive used for capturing and manipulating screenshots",
+  "homepage": "https://applitools.com",
+  "bugs": {
+    "url": "https://github.com/applitools/eyes.sdk.javascript1/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/applitools/eyes.sdk.javascript1.git",
+    "directory": "js/packages/image"
+  },
+  "license": "SEE LICENSE IN LICENSE",
+  "author": {
+    "name": "Applitools Team",
+    "email": "team@applitools.com"
+  },
+  "aliases": [
+    "screenshoter"
+  ],
```

Aha. I see what confused me. We use the first alias as the name of the job, so it says that `screenshoter` doesn't have `scripts` dependency when the `image` is the one that doesn't have it. 
```suggestion
  "aliases": [
    "image"
  ],
```